feat: rename auth48 in email template for final review (#1292)
ietf-tools/rpc: templates/rpc/mail/finalreview.txt

RFC Author(s):
--------------

Final Review for RFC-to-be {{ rfc_to_be.rfc_number }} <{{ rfc_to_be.draft.name }}>

> Your document has now entered Final Review. Once it has been reviewed and

Reviewer: Per Sandy's suggestions linked in #1290:

```suggestion
Your document is now available for Final Review (previously AUTH48). Once it has been reviewed and
```

approved by you and all coauthors, it will be published as an RFC.

> If an author is no longer available, there are several remedies

Reviewer 2: ```suggestion
If an author is no longer available, there are several remedies;
```

> available as listed in the FAQ (https://www.rfc-editor.org/faq/).

Reviewer 2: ```suggestion
see the Unavailable Authors section (https://authors.ietf.org/rfc-publication-process#unavailable-authors).
```
